_posts/articles/2017-03-18-pantheon-case-study.md

```diff
@@ -0,0 +1,69 @@
+---
+layout: post
+title: Case Study - Pantheon, Rachel Whitton
+excerpt: "In this use case, the Technical Content Editor at Pantheon, Rachel Whitton, describes their use of GitHub for documentation."
+modified: Sat Mar 18 09:50:27 CDT 2017
+categories: articles
+author: rachel_whitton
+tags: [case study, use case, github, docs, repos, web, content, workflows]
+image:
+  feature: stevensnodgrass-bolt.jpg
+  credit: Flickr stevensnodgrass
+  creditlink: https://flic.kr/p/8jKjE2
+comments: true
+share: true
+---
+
+Pantheon is a website management platform for Drupal and Wordpress. In this use case, their Technical Content Editor Rachel Whitton describes the use of GitHub for documentation as well as their workflow.
```

Comment:
WordPress* ;) [Folks are mighty picky about that capitol P](https://codex.wordpress.org/Function_Reference/capital_P_dangit)